lib/ar/sequence/schema_dumper.rb

```diff
@@ -36,18 +36,31 @@ def sequences(stream)
           next unless @connection.custom_sequence?(sequence_full_name)
 
           start_value = seq["start_value"]
+          minimum_value = seq["minimum_value"]
+          maximum_value = seq["maximum_value"]
+          cycle_option = seq["cycle_option"]
           increment = seq["increment"]
 
           options = []
 
-          if start_value && Integer(start_value) != 1
+          if start_value && Integer(start_value) != Integer(minimum_value)
```

Comment:
This corrects the logic to match the [postgres documentation](https://www.postgresql.org/docs/current/sql-createsequence.html). The default `start_value` is actually `minimum_value`, which defaults to `1`. Now that you can change `minimum_value` to things other than `1`, the logic has to change accordingly.